src/soda-js.coffee

```diff
@@ -87,7 +87,14 @@ expr =
   lt:  (column, literal) -> "#{column} < #{handleLiteral(literal)}"
   lte: (column, literal) -> "#{column} <= #{handleLiteral(literal)}"
   eq:  (column, literal) -> "#{column} = #{handleLiteral(literal)}"
-
+  
+# serialize object to querystring
+serialize = (obj) ->
+  str = []
+  for p of obj
+    if obj.hasOwnProperty(p)
```

Comment:
A quick CoffeeScript-ism you can incorporate — this line can be removed in favor of changing the previous one to `for own p of obj`.